otel/metrics.go

```diff
@@ -0,0 +1,25 @@
+package otel
+
+import (
+	sdkmetric "go.opentelemetry.io/otel/metric"
+)
+
+type Meter interface {
+	GetOriginalMeter() sdkmetric.Meter
```

Comment:
We don't need this `GetOriginalMeter()`. The interface is needed for the next PR to list all methods to create instruments
```suggestion
```